docs/resources/registry_namespace.md

```diff
@@ -36,6 +36,8 @@ The following arguments are supported:
 
 - `organization_id` - (Defaults to [provider](../index.md#organization_id) `organization_id`) The ID of the organization the registry is associated with.
 
+- `project_id` - (Defaults to [provider](../index.md#project_id) `project_id`) The ID of the project the registry is associated with.
```

Comment:
NIT: in the data-source, you use `registry` and in the resource your use `registry`, we should choose one word.